Doc/whatsnew/3.10.rst

```diff
@@ -1070,6 +1070,27 @@ Deprecated
   ``cache=shared`` query parameter.
   (Contributed by Erlend E. Aasland in :issue:`24464`.)
 
+* The following ``threading`` methods are now deprecated:
+
+  * ``threading.currentThread`` => :func:`threading.current_thread`
+
```

Comment:
It doesn't work without these empty lines?

Reply:
That's right, they all get mushed together into a single line